python/paddle/fluid/dygraph/dygraph_to_static/convert_builtins_func.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+from paddle.fluid import framework
+from paddle.fluid import core
+from paddle.fluid import unique_name
+
+
+def converted_len(var):
+    # return variable(length) from shape ops based on var.type
+    if isinstance(var, framework.Variable):
+        block = current_block(var)
+        out = create_new_var(block, 'int32', name='shape')
+        if var.type in [
+                core.VarDesc.VarType.LOD_TENSOR,
+                core.VarDesc.VarType.SELECTED_ROWS
+        ]:
+            # Note: Length of var may be known ahead of time in dygraph,
+            # but it probably represents batch size which can be variant.
+            # so we return a variable dynamically inferred from var.shape.
+            block.append_op(
```

Comment:
Here we are doing block-level operation, which means we are not only translating dygraph func into static func code. Add a simple doc comment to say that this function does block-level operation instead of source code level translating.

Reply:
Thanks，done.